subsys/bluetooth/host/hci_core.c

```diff
@@ -5980,6 +6014,37 @@ int bt_le_adv_stop(void)
 	return 0;
 }
 
+void bt_le_adv_resume(void)
+{
+	struct bt_conn *adv_conn = NULL;
+	int err;
+
+	if (IS_ENABLED(CONFIG_BT_PERIPHERAL) &&
+	    atomic_test_bit(bt_dev.flags, BT_DEV_ADVERTISING_CONNECTABLE)) {
+		adv_conn = bt_conn_add_le(bt_dev.adv_id, BT_ADDR_LE_NONE);
+		if (!adv_conn) {
+			return;
+		}
+
+		bt_conn_set_state(adv_conn, BT_CONN_CONNECT_ADV);
+	}
+
+	if (IS_ENABLED(CONFIG_BT_PRIVACY)) {
+		le_set_private_addr(bt_dev.adv_id);
+	}
+
+	err = set_advertise_enable(true);
+	if (err) {
+		if (IS_ENABLED(CONFIG_BT_PERIPHERAL) && adv_conn) {
+			bt_conn_set_state(adv_conn, BT_CONN_DISCONNECTED);
+		}
+	}
+
+	if (IS_ENABLED(CONFIG_BT_PERIPHERAL) && adv_conn) {
```

Comment:
Like in the other case where I commented on the same, I think the `IS_ENABLED()` could be removed here

Reply:
This function is now undef #ifdef CONFIG_BT_PERIPHERAL so removed all IS_ENABLED here.